app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  def show
+    user = User.find(params[:id])
+    render json: user
+  end
+
+  def create
+    user = User.create(user_params)
```

Comment:
Does this also persist the user to the db? Can invalid users be added to the db?

Reply:
I'm going to add some tests to check for this.